### PR TITLE
feat: [CDB-248]: Added Corporate Branding Colors

### PR DESCRIPTION
**Pull request by Harness-JKhattak**
### Summary

- Added new colors for specific Modules within the Harness platform (Continuous Delivery, Continuous Integration, Cloud Cost Management, Feature Flag Management, Service Reliability Management, Security Test Orchestration, Chaos Engineering)

**[>> See Brand Book for the colors that are used <<](https://www.figma.com/file/Id9M0XkxLuVX1bYse2tyo1/Brand-Book?node-id=28%3A1739)**

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/design-system/src/styled-props/background.css [hunk 1]
```diff
@@ -506,6 +506,34 @@
     background: var(--primary-bg);
   }
 
+  &.background-corporateCdBackground {
```
Comment (knagurski) [inline packages/design-system/src/styled-props/background.css:509]: You can probably drop the `Background` suffix on these

File: packages/design-system/src/styled-props/border/border.css [hunk 2]
```diff
@@ -281,6 +281,34 @@
     border-color: var(--magenta-500);
   }
 
+  &.border-color-corporateCdBorder {
```
Comment (knagurski) [inline packages/design-system/src/styled-props/border/border.css:284]: And the `Border` suffix on these too

File: packages/uicore/src/styles/colors.css [hunk 3]
```diff
@@ -163,6 +163,43 @@
...
+  /* Chaos Engineering */
+  --corporate-ce-900: #350018;
+  --corporate-ce-800: #68002d;
+  --corporate-ce-700: #9a0041;
+  --corporate-ce-600: #cd0056;
+  --corporate-ce-500: #ff006a;
+  --corporate-ce-400: #ff368a;
+  --corporate-ce-300: #ff6cab;
+  --corporate-ce-200: #ffa1cb;
+  --corporate-ce-100: #fff1f7;
```
Comment (knagurski) [inline packages/uicore/src/styles/colors.css:201]: Do we need these? I thought we just needed a foreground/background/border per module?

File: packages/design-system/src/styled-props/background.css (hunk 1 above)
Comment (Harness-JKhattak) [reply packages/design-system/src/styled-props/background.css:509]: That's fair, I just wanted to match the naming convention from all the other colors above

File: packages/uicore/src/styles/colors.css (hunk 3 above)
Comment (Harness-JKhattak) [reply packages/uicore/src/styles/colors.css:201]: In the brand Color document, CE is the only module that has a range of colors:

Screenshot 2022-07-13 at 16 02 05

Although I could keep it in line with the other colors for now until we actually need those other colors

**Comment by Harness-JKhattak** [comment]
Closing due to this being moved into harness-ui-core -> https://github.com/harness/harness-core-ui/pull/10018

**Pull request closed without merging**